main/admin/resume_session.php

```diff
@@ -10,6 +10,12 @@
 $cidReset = true;
 require_once '../inc/global.inc.php';
 
+$advancedSessionsPluginFilePath = api_get_path(PLUGIN_PATH) . 'advancedsessions/src/AdvancedSessionsPlugin.class.php';

```

Comment:
You don't need to do this. Just run an composer update and it will add the class in the autoload.